src/stdio.rs

```diff
@@ -0,0 +1,124 @@
+use std::{io, time};
+
+pub struct Stdio;
+
+impl serialport::SerialPort for Stdio {
```

Comment:
A successful `SerialPortBuilder::open` returns `Box<dyn SerialPort>` however from what I have seen so far (might have missed something), we only seem to need `io::Read` and `io::Write`, if we could get a trait object of that we would not have to implement `SerialPort` for `Stdio`. Not sure if that is doable though...

Reply:
implementing `SerialPort` for `Stdio` seems to be a bit hacky indeed. Could we create a new struct called `Interface` that implements `io::Read` and `io::Write` and then two builder functions: `from_serial()` and `from_read()` ?
Not sure how to handle the configuration of the serial port then (baud, parity etc)..